src/components/MarkdownViewer.test.tsx

```
@@ -1,330 +1,200 @@
-        Deselect the first risk
-      </button>
-
-      <input
-        data-testid="prompt-input"
-        value={props.chatPrompt}
-        onChange={(e) => props.setChatPrompt(e.target.value)}
-      />
-      <button data-testid="send-prompt-btn" onClick={props.handleSendPrompt}>
-        Send Prompt
-      </button>
+  default: (p: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)